surveys/2024-annual-survey/translations/ru.md

```diff
@@ -145,46 +145,46 @@ Type: select all that apply
 
 - Текущая стабильная версия
 - Предыдущая стабильная версия
-- Стабильная версия Rust 1.67 или новее
-- Стабильная версия Rust старше 1.67
-- Бета версия
-- Последняя ночная версия
-- Определённая ночная версия
-- Пользовательская версия
+- Стабильная версия Rust 1.75 или новее
+- Стабильная версия Rust старше 1.75
+- Бета-версия
+- Последняя ночная (nightly) версия
+- Определённая ночная (nightly) версия
+- Кастомная версия (fork)
 - Я не знаю
 - Другое
 
-### Если используете ночную версию, почему?
+### Если используете ночную (nightly) версию, почему?
 
 Type: select all that apply
 
-- Я не использую ночную версию
+- Я не использую ночную (nightly) версию
```

Comment:
`nightly` already mentioned above, so it's better to avoid repeating it once again in English here.